src/main/java/org/opensearch/knn/common/KNNVectorUtil.java

```diff
@@ -62,17 +60,4 @@ public static int[] intListToArray(final List<Integer> integerList) {
         }
         return intArray;
     }
-
-    /**
-     * Iterates vector values once if it is not at start of the location,
-     * Intended to be done to make sure dimension and bytesPerVector are available
-     * @param vectorValues
-     * @throws IOException
-     */
-    public static void iterateVectorValuesOnce(final KNNVectorValues<?> vectorValues) throws IOException {
```

Comment:
Why we have moved this to KNNCodecUtil?

Reply:
i feel that was not the right place for this util. As the function was used only in codec so I moved it.